### ISSUE TITLE
Missing objects when taken from clear to Snow

### ISSUE THREAD

**Issue by code-Assasin** [junyanz/pytorch-CycleGAN-and-pix2pix#436]
Hey, I am trying to train the cycle-Gan on from clear weather street images(A) to snowy street images(B). In the transition from A to B, the buildings present in the images from A seem to be disappearing, while the reconstruction seems fine again. Even while transitioning from B to A parts of objects seem to be missing. Any clue why this is happening?
Dataset used: http://data.nvision2.eecs.yorku.ca/JAAD_dataset/
Epochs trained: 170
Note: The images are totally unpaired. 

<img width="1058" alt="screen shot 2018-11-13 at 2 47 22 am" src="https://user-images.githubusercontent.com/15107872/48375663-a0d94a80-e6ee-11e8-9c4d-07b386575364.png">
<img width="843" alt="screen shot 2018-11-13 at 12 46 09 am" src="https://user-images.githubusercontent.com/15107872/48375664-a0d94a80-e6ee-11e8-9669-315e44ce97cf.png">

**Comment by junyanz** [comment]
I guess somehow, there might be more trees in snowy days than in clear days. Maybe you can try to do cyclegan in the cropped patches (e.g., if your original image size is 256, you can set `loadSize 256 fineSize 128` or `loadSize 256 fineSize 96`). You can also use the identity [loss](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/docs/qa.md#what-is-the-identity-loss-322-373-362).

**Comment by code-Assasin** [comment]
It seems what you have pointed about the dataset is true. The resolution of my input images is very high(1920 × 1080). What would be preferable values for load and finesize for good results? Also, would it be a good idea to use `--no_flip` during training?

**Comment by code-Assasin** [comment]
Are all the patches created used for data augmentation or only one random patch is selected?

**Comment by code-Assasin** [comment]
Thanks a lot!! Your immediate responses helped a lot!!
